packages/charts/react-charting/src/components/PieChart/Arc/Arc.styles.ts

```diff
@@ -1,11 +1,19 @@
 import { IArcProps, IArcStyles } from './Arc.types';
-import { DefaultPalette, ITheme } from '@fluentui/react/lib/Styling';
+import { DefaultPalette, ITheme, HighContrastSelectorBlack } from '@fluentui/react/lib/Styling';
 export const getStyles = (props: IArcProps, theme: ITheme | undefined): IArcStyles => {
   const { color } = props;
   return {
     arcRoot: { fill: color },
     arcRootFocussed: { fill: color, stroke: theme?.palette.black || DefaultPalette.black, strokeWidth: 2 },
     arc: { outline: 'none' },
-    arcText: { fill: theme?.palette.black || DefaultPalette.black, outline: 'none' },
+    arcText: {
+      fill: theme?.semanticColors.bodyText || DefaultPalette.black,
+      outline: 'none',
+      selectors: {
+        [HighContrastSelectorBlack]: {
+          fill: 'rgb(255, 255, 255)',
```

Comment:
Let's replace this with the `CanvasText` [system color](https://developer.mozilla.org/en-US/docs/Web/CSS/system-color).